Junk4Dollars-app/Auctions/AuctionListView.swift

```diff
@@ -0,0 +1,4 @@
+import UIKit
+public protocol AuctionListView: UIViewController {
+    func refreshAuctions(_ auctions: [Auction]) -> Void
```

Comment:
I would remove the underscore. It is definitely idiomatic Swift to leverage named parameters. Personally, I also feel that `refresh(actions: [Auction])` also reads better.